### PR TITLE
Use the fetch JS API to get random facts

### LINKED ISSUE #5: Add server side quote generator
Use a java servlet and the JS fetch API to randomly generate a quote on the random page.

### PR DESCRIPTION
**Pull request by CarlosM10011**
This pull request is the client side for issue #5, which is to migrate the fact generator to a java servlet.

closes #5

### CODE REVIEW

File: portfolio/src/main/webapp/script.js
Comment (hzimz) [inline portfolio/src/main/webapp/script.js]: It's good practice to wrap this in a try-catch block to handle the await call failing.

**Review by hzimz — CHANGES_REQUESTED** [review]

**Review by tinagrasso — APPROVED** [review]
Please wait for Hussein's approval before merging.

**Review by hzimz — APPROVED** [review]

File: portfolio/src/main/webapp/script.js
Comment (hzimz) [inline portfolio/src/main/webapp/script.js]: Sorry, move the initial await outside the try-catch block

**Review by hzimz — CHANGES_REQUESTED** [review]

**Review by hzimz — APPROVED** [review]

**Pull request merged**
